static/app/components/charts/optionSelector.tsx

```diff
@@ -118,6 +142,7 @@ const StyledDropdownBubble = styled(DropdownBubble)<{
   minWidth?: number;
 }>`
   display: ${p => (p.isOpen ? 'block' : 'none')};
+  overflow: visible;
```

Comment:
So that the hover doesn't get hidden in the dropdown, eg.
<img width="467" alt="image" src="https://user-images.githubusercontent.com/4205004/122835430-506ef400-d2be-11eb-9094-749db6615db9.png">